src/QsFmt/Formatter/Rules.fs

```diff
@@ -160,6 +160,61 @@ let forParensUpdate =
             }
     }
 
+/// <summary>
+/// Make sure that a <see cref="SequenceItem"/> contains a comma.
+/// </summary>
+let ensureComma (item: 'a SequenceItem) =
+    match item.Comma with
+    | None -> { Item = item.Item; Comma = { Prefix = []; Text = "," } |> Some }
+    | Some _ -> item
+
+/// <summary>
+/// Prepends the <paramref name="parameters"/> with an ellipsis <see cref="Terminal"/> item if it does not already contain one.
+/// </summary>
+let ensureEllipsis (parameters: Terminal Tuple) =
+    let ellipsis nspace =
+        { Prefix = [ spaces nspace ]; Text = "..." }
+
+    let ellipsisItem nspace =
+        { Item = ellipsis nspace |> Some; Comma = None }
+
+    { parameters with
+        Items =
+            match parameters.Items with
+            // Replace, e.g., `body ()` with `body (...)`
+            | [] -> [ ellipsisItem 0 ]
+            // Replace, e.g., `controlled (q)` with `controlled (q, ...)`
+            | [ x ] ->
+                match Option.get(x.Item).Text with
+                | "..." -> [ x ]
+                | _ -> [ ensureComma x; ellipsisItem 1 ]
+            | _ -> parameters.Items
+    }
+
+let specializationUpdate =
```

Comment:
The documentation for the Formatter has recently been updated to include the rule set that is run. Could you also add this rule to the QsFmt's README.md file, under the section "Rules that are run"? [Link to Section](https://github.com/yjt98765/qsharp-compiler/blob/specialization/src/QsFmt/README.md?plain=1#L52)

Reply:
I have added the first version to that section. It is challenging to summarize this update rule in a sentence, though. Please modify it if you have a better idea!

Reply:
Looks good, thanks!